linux/shim/init.go

```diff
@@ -237,31 +237,30 @@ func (p *initProcess) SetExited(status int) {
 }
 
 func (p *initProcess) Delete(context context.Context) error {
-	status, err := p.Status(context)
-	if err != nil {
-		return err
-	}
-	if status != "stopped" {
```

Comment:
It does.  And we added this check in the client when we implemented the Force stop on Delete